src/CTA.Rules.Actions/ActionHelpers/ServerConfigMigrate.cs

```diff
@@ -184,7 +185,7 @@ private void HandleHandlers(XElement httpHandlerElement)
                 {
                     var handlerName = GetAttributeValue(handler, Constants.Type);
                     var handlerPath = GetAttributeValue(handler, Constants.PathAttribute, "*");
-                    if(!ServerConfigTemplates.UnsupportedHandlers.Contains(handlerName.Trim()))
+                    if(handlerName is not null && !ServerConfigTemplates.UnsupportedHandlers.Contains(handlerName.Trim()))
```

Comment:
`handlerPath` can also be null

Reply:
I will fix this. I thought `string.Format` could handle null inputs, but it cannot.